src/transformers/cache_utils.py

```diff
@@ -1137,7 +1135,7 @@ def __init__(
         device: torch.device = None,
         dtype: torch.dtype = torch.float32,
         max_batch_size: Optional[int] = None,
-        layer_device_map: Optional[Dict[int, Union[str, torch.device, int]]] = None,
```

Comment:
Instead of `**kwargs`, let's deprecate to then delete (and use the deprecation decorator)